www/index.html

```diff
@@ -6,7 +6,8 @@
         <title>Kiwix</title>
         <meta name="description" content="Offline Wikipedia reader">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: https://download.kiwix.org https://master.download.kiwix.org https://browser-extension.kiwix.org https://kiwix.github.io 'unsafe-inline' 'unsafe-eval'; frame-src 'self' moz-extension: chrome-extension:; object-src 'none';">
+        <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: https://download.kiwix.org https://master.download.kiwix.org https://browser-extension.kiwix.org https://kiwix.github.io 'unsafe-inline' 'unsafe-eval'; frame-src 'self' moz-extension: chrome-extension: https://library.kiwix.org/ download.kiwix.org *.mblibrary.info; object-src 'none';">
```

Comment:
already tried `https://library.kiwix.org/` and `library.kiwix.org` but i will retry

Reply:
I've been thinking that you might be able to turn this off completely, programmatically, when you load the library. See https://stackoverflow.com/questions/18982228/how-to-add-meta-tag-in-javascript . It seems it can be manipulated.

Reply:
But try the `*` solution first!